src/cljc/matcher_combinators/matchers.cljc

```diff
@@ -74,6 +74,12 @@
   [pred]
   (core/->PredMatcher pred (str "predicate: " pred)))
 
+(defn not-matcher
```

Comment:
is `not` a better name? or can someone think of something nicer?

Reply:
I like `not` or `mismatch` (since that's what's in the error message)